app/models/disability_compensation_submission.rb

```diff
@@ -11,4 +11,8 @@ class DisabilityCompensationSubmission < ActiveRecord::Base
     class_name: 'AsyncTransaction::EVSS::VA526ezSubmitTransaction',
     foreign_key: 'va526ez_submit_transaction_id'
   )
+
+	def complete
```

Comment:
is this method to cover existing records? If so I think we're ok with just the default migration